src/templates/analytics/analytics.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block extra_head %}
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/fomantic-ui/2.7.5/semantic.min.css">
```

Comment:
I think this is duplicate to:
https://github.com/codalab/competitions-v2/blob/9525aebba50e9f1d5d37faf4e2461c890a284e59/src/templates/base.html#L15

Reply:
Removed. Thank you for recognizing this.